ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -28,10 +28,13 @@ tests:
       CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:osc-test-fbc-on-push-pzw5x-build-image-index
       CATALOG_SOURCE_NAME: brew-catalog
       CUSTOM_AZURE_REGION: eastus
+      ENABLE_MUST_GATHER: "true"
       EXPECTED_OSC_VERSION: 1.10.0
       EXPECTED_TRUSTEE_VERSION: 0.4.1
       INSTALL_KATA_RPM: "true"
       KATA_RPM_VERSION: 3.17.0-3.rhaos4.19.el9
+      MUST_GATHER_IMAGE: registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest
+      MUST_GATHER_ON_FAILURE_ONLY: "true"
```

Comment:
As I commented on previous commit, I think it'd be nice to set the defaults to cover most cases and leave our templates shorter. They contain a lot of variables already, what do you think?